tasks/updateconstants.js

```diff
@@ -941,8 +941,12 @@ async function start() {
           scripts = { ...scripts, ...heroAbs };
         }
 
+        // Previous data for deprecated facets
+        const oldHeroAbilities = require("../build/hero_abilities");
```

Comment:
Does this make the build dependent on having an existing hero abilities file already? e.g. we can't do a "clean" build

Ideally we would move the deprecated data to a manual file in the json directory

Reply:
A clean build would currently not get the name and description of a deprecated facet. I'll look into getting the strings from game files instead of datafeed.